src/NuGet.Core/NuGet.Commands/Strings.resx

```diff
@@ -511,4 +511,40 @@ For more information, visit http://docs.nuget.org/docs/reference/command-line-re
     <value>Clearing NuGet Temp cache: {0}</value>
     <comment>{0} : Temp cache path</comment>
   </data>
+  <data name="Log_RestoreNoOpDGChanged" xml:space="preserve">
+    <value>The restore inputs for '{0}' have changed. Continuing restore.</value>
+  </data>
+  <data name="Log_RestoreNoOpFinish" xml:space="preserve">
+    <value>The restore inputs for '{0}' have not changed. No further actions are required to complete the restore.</value>
+  </data>
+  <data name="Log_SkippingLockFile" xml:space="preserve">
+    <value>Lock file has not changed. Skipping lock file write. Path: {0}</value>
```

Comment:
Would you update this to say ``assets file`` instead of lock file? ``assets`` will be correct for 95% of the scenarios now, where as lock will be correct for 5%.  I think it's okay to have a single message for both.